mobile-app/app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -118,17 +116,21 @@ export function OceanInterface (): JSX.Element | null {
             transaction.onBroadcast()
           }
           let title
+          let transactionStatusCode
           try {
             await waitForTxConfirmation(transaction.tx.txId, client, logger)
-            title = 'Transaction completed'
+            title = 'Transaction confirmed'
+            transactionStatusCode = 200
```

Comment:
Let's make this an enum

Reply:
@thedoublejay done check if its up to standard haha